code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -106,21 +110,22 @@
 	// Hijack objective.
 	if(prob(10) && !(locate(/datum/objective/hijack) in owner.get_all_objectives()))
 		add_antag_objective(/datum/objective/hijack)
+		organisation = new /datum/antag_org/syndicate/gorlex
 		return // Hijack should be their only objective (normally), so return.
 
-	// Will give normal steal/kill/etc. type objectives.
-	for(var/i in 1 to GLOB.configuration.gamemode.traitor_objectives_amount)
-		forge_single_human_objective()
+	var/i = 0
+	//If our org has a forced objective, give it to us guaranteed.
+	if(organisation && organisation.forced_objective)
+		add_antag_objective(organisation.forced_objective)
+		i++
 
-	var/can_succeed_if_dead = TRUE
-	for(var/objective in owner.get_all_objectives())
-		var/datum/objective/O = objective
-		if(!O.martyr_compatible) // Check if our current objectives can co-exist with martyr.
-			can_succeed_if_dead  = FALSE
-			break
+	// Will give objectives from our org or random objectives.
+	while(i < GLOB.configuration.gamemode.traitor_objectives_amount)
+		forge_single_human_objective()
+		i++
 
 	// Give them an escape objective if they don't have one already.
-	if(!(locate(/datum/objective/escape) in owner.get_all_objectives()) && (!can_succeed_if_dead || prob(80)))
+	if(!(locate(/datum/objective/escape) in owner.get_all_objectives()))
```

Comment:
Undocumented change.

Reply:
This is leftover code from when we had the martyr objective as an option instead of escape, there's no change incurred from removing it.